pom.xml

```diff
@@ -85,13 +72,14 @@
         </pluginManagement>
     </build>
     <modules>
-        <module>lib</module>
         <module>api</module>
         <module>demo</module>
         <module>app</module>
+        <module>lib</module>
```

Comment:
Why? This seems backwards.

Reply:
Still true. Does not really matter (Maven will reorder the build by topological sort) but misleading.